app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java

```diff
@@ -557,6 +577,51 @@ public String convertIdeographicSpaceToLatinSpace(String source) {
 
     }
 
+    private void updateAddButtonVisibility() {
```

Comment:
Could you please add some Javadoc?

Reply:
Sure ! and Thanks for reporting, I have added javadoc in newer commit https://github.com/commons-app/apps-android-commons/pull/5464/commits/f425eadf17b6d18f861dc95c66a7d71f6085a2c7